### PR TITLE
Project setup

### PR DESCRIPTION
**Pull request by serengia**
Hello. Kindly review my request to merge `project-setup` branch to `development` branch.
The branch contains the following implementations:
- Routing implementation
- Components implementation
- Basic styling

### CODE REVIEW

File: src/pages/CategoriesPage.js [hunk 1]
```diff
@@ -1,7 +1,13 @@
...
 
 function CategoriesPage() {
-  return <div>CategoriesPage</div>;
+  return (
+    <div>
+      <div className="categories-wrapper row">
+        <h1>Categories</h1>
+      </div>
+    </div>
+  );
```
Comment (roshan-bajgain) [inline src/pages/CategoriesPage.js]: - [x] According to the project requirement on your categories page you should display a button "Check status" not `h1` tag. please changes it

File: src/App.js [hunk 2]
```diff
@@ -1,13 +1,24 @@
...
 import CategoriesPage from "./pages/CategoriesPage";
 
+const router = createBrowserRouter(
+  createRoutesFromElements(
+    <Route path="/" element={<RootLayout />}>
+      <Route path="books" element={<BooksPage />} />
+      <Route path="categories" element={<CategoriesPage />} />
+    </Route>
+  )
+);
```
Comment (roshan-bajgain) [inline src/App.js]: - [x] [Optional] I would suggest you please use `BooksPage` as the home page. When users visit the browser for the first time then they will see the list of books.

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @serengia, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Professional README file. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by serengia** [comment]
Thank you for your feedback. I will do the suggested changes as soon as possible.

**Review by danmainah — APPROVED** [review]
Hi @serengia , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ No linter errors
✔️ Gitflow was correctly used
✔️  Good design

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by serengia** [comment]
Thank you.

**Pull request merged**